README.md

```diff
@@ -58,11 +60,32 @@ L-IPAM requires following [IAM policy](https://docs.aws.amazon.com/IAM/latest/Us
   There are 2 components:
   
   * [CNI Plugin](https://kubernetes.io/docs/concepts/cluster-administration/network-plugins/#cni), which will wire up host's and pod's network stack when called.
-  * L-IPAM, which is a long running node-Local IP Address Management (IPAM) daemon, is responsible for:
+  * `L-IPAMD`, which is a long running node-Local IP Address Management (IPAM) daemon, is responsible for:
     * maintaining a warm-pool of available IP addresses, and
     * assigning an IP address to a Pod.
 
-The details can be found in [Proposal: CNI plugin for Kubernetes networking over AWS VPC](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/cni-proposal.md)
+The details can be found in [Proposal: CNI plugin for Kubernetes networking over AWS VPC](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/cni-proposal.md).
+
+[Troubleshooting Guide](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/troubleshooting.md) provides tips on how to debug and troubleshoot CNI.
+
+### Notes
+
+`L-IPAMD`(aws-node daesomSet) running on every worker node requires access to kubernetes API server.  If it can **not** reach kubernetes API server, ipamD will exit and CNI will not able to get any IP address for Pods.  Here is a way to confirm if `L-IPAMD` have access to kubernetes API server.
```

Comment:
Replace `Here is a way to confirm if L-IPAMD have access to kubernetes API server.` --> 
`Here is a way to confirm if L-IPAMD has access to the kubernetes API server.`